CmrUI/Navigation/CmrContainerVC.swift

```diff
@@ -0,0 +1,185 @@
+//
+//  CmrContainerVC.swift
+//  CmrUI
+//
+
+import UIKit
+
+class CmrContainerVC: UIViewController {
+    
+    private var tab: ContainerTab?
+    private var currentVC: UIViewController?
+    
+    var alert: UIAlertController? // Will implement it later, maybe it will replaced by PXAlertView
+    private lazy var activityIndicator = UIActivityIndicatorView(activityIndicatorStyle: .gray) // Will move it to separate class later
```

Comment:
@vmsavka Do you have any way to mark those things that you plan on doing later? We usually use TODO and then pass over all the TODO items.